content/en/docs/concepts/configuration/overview.md

```diff
@@ -87,7 +87,7 @@ The [imagePullPolicy](/docs/concepts/containers/images/#updating-images) and the
 - `imagePullPolicy: Never`: the image is assumed to exist locally. No attempt is made to pull the image.
 
 {{< note >}}
-To make sure the container always uses the same version of the image, you can specify its [digest](https://docs.docker.com/engine/reference/commandline/pull/#pull-an-image-by-digest-immutable-identifier), for example, replace `image-name:latest` with `image-name@sha256:45b23dee08af5e43a7fea6c4cf9c25ccf269ee113168c19722f87876677c5cb2`. The digest uniquely identifies a specific version of the image, so it is never updated by Kubernetes unless you change the digest value.
+To make sure the container always uses the same version of the image, you can specify its [digest](https://docs.docker.com/engine/reference/commandline/pull/#pull-an-image-by-digest-immutable-identifier), replace `<image-name>:<tag>` with `<image-name>@<digest>` (e.g. `image@sha256:45b23dee08af5e43a7fea6c4cf9c25ccf269ee113168c19722f87876677c5cb2`). The digest uniquely identifies a specific version of the image, so it is never updated by Kubernetes unless you change the digest value.
```

Comment:
Thanks @aelij . Looks good. One nit, add back `, for example,` to the sentence or start a new sentence with `For example,`:

To make sure the container always uses the same version of the image, you can specify its ...
For example, replace `<image-name>:<tag>` with `<image-name>@<digest>` (e.g. `image@sha256:45b23dee08af5e43a7fea6c4cf9c25ccf269ee113168c19722f87876677c5cb2`) ...

Reply:
But with the placeholders, that's no longer an example, it's just the syntax :) I specifically added an example (indicated by *e.g.* in brackets).

Reply:
If you can [avoid Latin phrases](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases), that's even better.

Reply:
@sftim Done